test_setup.py

```diff
@@ -5,13 +5,17 @@
 if __name__=="__main__" :
     
     
-    # Create a test environment for use later
-    board_size = 5
+    # # Create a test environment for use later
+    board_size = 15
     configuration = {"size": board_size, "startingHalite": 1000}
-    environment = make("halite", configuration=configuration)
-    agent_count = 2
-    environment.reset(agent_count)
-    state = environment.state[0]
 
-    actions = base_agent(state.observation, environment.configuration )
+    from kaggle_environments import make
+    env = make("halite", debug=True, configuration=configuration)
+    env.run(["agent.py", "agent.py", "agent.py", "random"])
```

Comment:
This plays the game between 3 instances of agent.py and random

Using this we can create our experience memory for training !